nengo/builder/transforms.py

```diff
@@ -158,6 +170,10 @@ def __init__(self, W, X, Y, conv, tag=None):
         self.reads = [W, X]
         self.updates = []
 
+    @property
+    def transpose(self):
```

Comment:
I'd use `is_transpose` here (just so that there's no confusion with something like `ndarray.transpose`, which actually transforms the object to be a transposed version).